spec/factories/comments.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+
+  factory :comment do
+    content "This is a sample comment."
+    association :commentable, factory: :user
```

Comment:
Check out [Traits](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#traits) for setting up different commentable types, so you could do something like `create :comment, :for_post` or `build :comment, :for_comment`.